package.json

```diff
@@ -25,8 +25,7 @@
     "react-table": "^6.6.0",
     "react-tooltip": "^3.4.0",
     "topojson-client": "^3.0.0",
-    "victory": "^0.22.2",
-    "victory-core": "^20.2.0"
```

Comment:
Don't we need victory-core?

Reply:
no. `victory-core` is `victory` dependency. we don't need it here.